aws-kendra-index/src/main/java/software/amazon/kendra/index/ListHandler.java

```diff
@@ -24,7 +32,7 @@ public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         final ListIndicesRequest listIndicesRequest = Translator.translateToListRequest(request.getNextToken());
 
         // STEP 2 [TODO: make an api call]
```

Comment:
I know its not from your changes, but can we remove these TODO comments?

Reply:
sure!